diff -up contour-0.1.1/CMakeLists.txt.find-qtmobility contour-0.1.1/CMakeLists.txt
--- contour-0.1.1/CMakeLists.txt.find-qtmobility	2011-09-28 19:05:15.000000000 +0200
+++ contour-0.1.1/CMakeLists.txt	2011-11-08 11:43:56.988493119 +0100
@@ -1,6 +1,6 @@
 project(contour)
 
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${nepomuk_kde_SOURCE_DIR}/cmake)
+set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules ${CMAKE_MODULE_PATH} ${nepomuk_kde_SOURCE_DIR}/cmake)
 
 # set the version to be used by ContourConfig.cmake and the lib version
 # ===============================================================================================
